### DISCUSSION TITLE
error has occurring in Pydantic model.

### DISCUSSION THREAD

**Discussion by harshalkumar-ishi** [pydantic/pydantic#7240] [Projects using Pydantic]
Regex parse error while giving pattern to an field of model.

look-around, including look-ahead and look-behind, is not supported 

SchemaError exception has ocuring while giving follwing pattern: **'^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!#%*?&]+$'**

error is as follows:

**Error building "model" validator:
  SchemaError: Error building "model-fields" validator:
  SchemaError: Field "new_password1":
  SchemaError: Error building "str" validator:
  SchemaError: regex parse error:
    ^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!#%*?&]+$
     ^^^
error: look-around, including look-ahead and look-behind, is not supported**

**Comment by samuelcolvin** [comment]
duplicate of https://github.com/pydantic/pydantic/issues/7058
